nym-vpn-app/src-tauri/src/sentry.rs

```diff
@@ -29,9 +31,9 @@ pub fn init(os: &OsInfo) -> Option<ClientInitGuard> {
         #[cfg(target_os = "linux")]
         {
             scope.set_tag("display_server", os.display_server.as_ref());
-            scope.set_tag("gpu", os.gpu.as_ref());
```

Comment:
Please restore this, gpu type is a useful info for Linux, since it as impact on app rendering and for some combination (Wayland + Nvidia) it is the source of known issues we need to keep tracking